src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleSheetsArray = sourceString
+    .split('\n')
+    .filter((value) => value.length > 3)
+    .map((value) => value.replace(/;/g, ''))
+    .map(value => value.trim())
+    .map((value) => value.split(':'))
+    .sort((value1, value2) =>
```

Comment:
There's no need to sort the array, CSS rules order doesn't matter in an object representation.